util/src/main/java/tech/pegasys/teku/util/config/Constants.java

```diff
@@ -116,7 +116,7 @@
 
   // Teku Networking Specific
   public static final int VALID_BLOCK_SET_SIZE = 1000;
-  public static final int VALID_ATTESTATION_SET_SIZE = 1000;
+  public static final int VALID_ATTESTATION_SET_SIZE = 21000;
```

Comment:
I take that back, this appears to have been pushed to the wrong PR. :)

Reply:
definitely the wrong PR...